### PR TITLE
Automate test case to Enable and Delete a Kafka Stream (ods 242)

### PR DESCRIPTION
**Pull request by bdattoma**
The following PR wants to cover ODS-242 for testing the creation of a Kafka Stream starting from the Dashboard. It extends ODS-242 to include the deletion of the stream

### CODE REVIEW

File: tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot [hunk 1]
```diff
@@ -38,8 +38,7 @@ Verify User Is Able to Create And Delete a Kafka Stream
   Wait Until Keyword Succeeds    300  1  Check Stream Status  Ready
   Delete Kafka Stream Instance  stream_name=${stream_name_test}  stream_owner=${SSO.USERNAME}
   Wait Until Keyword Succeeds    300  1  Page Should Contain    No results found
-  Delete Configmap    name=rhosak-validation-result  namespace=redhat-ods-applications
-
+  Delete    kind=ConfigMap  name=rhosak-validation-result  namespace=redhat-ods-applications
 
 ** Keywords ***
```
Comment (tarukumar) [inline tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot:43]: i think we can add one more (*) to maintain consistency.

Comment (bdattoma) [reply tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot:43]: slipped away. Thanks, I fix it

**Comment by bdattoma** [comment]
Tested using Jenkins PR test job [#232](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/232/TestReport/)

This TC has been successfully executed

File: tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot (hunk 1 above)
Comment (bdattoma) [reply tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot]: done

**Review by vasukulkarni — APPROVED** [review]

**Review by vasukulkarni — APPROVED** [review]

**Pull request merged**